_config/cache.yml

```diff
@@ -7,7 +7,6 @@ SilverStripe\Core\Injector\Injector:
     constructor:
       args:
         directory: '`TEMP_PATH`'
-        version: null
```

Comment:
The `version` is effectively a flush mechanism - if the version changes, all cache with the previous version is invalidated.

Given this config isn't available when instantiating manifest cache, I've moved this to being an environment variable. In practice I don't think anyone would ever bother setting it since Silverstripe CMS has so many ways to flush cache.